ts/src/digifinex.ts

```diff
@@ -2952,31 +2965,57 @@ export default class digifinex extends Exchange {
      * @method
      * @name digifinex#transfer
      * @description transfer currency internally between wallets on the same account
+     * @see https://docs.digifinex.com/en-ww/spot/v3/rest.html#transfer-assets-among-accounts
+     * @see https://docs.digifinex.com/en-ww/swap/v2/rest.html#accounttransfer
      * @param {string} code unified currency code
      * @param {float} amount amount to transfer
-     * @param {string} fromAccount account to transfer from
-     * @param {string} toAccount account to transfer to
+     * @param {string} fromAccount 'spot', 'swap', 'margin', 'OTC' - account to transfer from
+     * @param {string} toAccount 'spot', 'swap', 'margin', 'OTC' - account to transfer to
      * @param {object} [params] extra parameters specific to the exchange API endpoint
      * @returns {object} a [transfer structure]{@link https://docs.ccxt.com/#/?id=transfer-structure}
      */
     async transfer (code: string, amount: number, fromAccount: string, toAccount:string, params = {}): Promise<TransferEntry> {
         await this.loadMarkets ();
         const currency = this.currency (code);
+        const currencyId = currency['id'];
         const accountsByType = this.safeValue (this.options, 'accountsByType', {});
         const fromId = this.safeString (accountsByType, fromAccount, fromAccount);
         const toId = this.safeString (accountsByType, toAccount, toAccount);
-        const request: Dict = {
-            'currency_mark': currency['id'],
-            'num': this.currencyToPrecision (code, amount),
-            'from': fromId, // 1 = SPOT, 2 = MARGIN, 3 = OTC
-            'to': toId, // 1 = SPOT, 2 = MARGIN, 3 = OTC
-        };
-        const response = await this.privateSpotPostTransfer (this.extend (request, params));
-        //
-        //     {
-        //         "code": 0
-        //     }
-        //
+        const request = {};
+        const fromSwap = (fromAccount === 'swap');
+        const toSwap = (toAccount === 'swap');
+        let response = undefined;
+        const amountString = this.currencyToPrecision (code, amount);
+        if (fromSwap || toSwap) {
+            if ((fromId !== '1') && (toId !== '1')) {
+                throw new ExchangeError (this.id + ' transfer() supports transferring from and to swap account between spot and swap only');
```

Comment:
@yzh-pelle This error message is a bit confusing 😅

Reply:
@carlosmiei 
I don't know how to word it better 😅
Any ideas?

Reply:
@yzh-pelle  Maybe just a few commas missing and minor adjustments, like 

```
transfer() supports transferring from, spot->swap, spot->margin and swap->spot only.
```

Reply:
```
transfer() supports transferring between spot and swap, spot and margin, spot and OTC only
```
Is it good?

Reply:
@yzh-pelle yess